docs/examples/metrics/views/disable_default_views.py

```diff
@@ -0,0 +1,49 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import time
+
+from opentelemetry.metrics import get_meter_provider, set_meter_provider
+from opentelemetry.sdk.metrics import MeterProvider
+from opentelemetry.sdk.metrics.export import (
+    ConsoleMetricExporter,
+    PeriodicExportingMetricReader,
+)
+from opentelemetry.sdk.metrics.view import DropAggregation, SumAggregation, View
+# disable default views
+# Create a view matching the counter instrument `mycounter`
+disable_default_views = View(instrument_name="*", aggregation=DropAggregation())
```

Comment:
Wouldn't it be more accurate to call this scenario `disable_default_aggregation`? Views are simply the mechanism we use to configure metrics, not the actual thing we are changing.

Reply:
@nanfuka this is still not addressed. I agree that `disable_default_aggregation` would be a better choice of naming.